gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -147,6 +147,14 @@ validate (GisPasswordPage *page)
   verify = gtk_entry_get_text (GTK_ENTRY (priv->confirm_entry));
 
   pw_strength (password, NULL, priv->username, &hint, &long_hint, &strength_level);
+
+    /*
+     * If the password is not empty but it's strength is 0, show a
+     * red bar instead of nothing.
+     */
+    if (gtk_entry_get_text_length (GTK_ENTRY (priv->password_entry)) > 0)
```

Comment:
Nit: this code is not indented the same as the surrounding code, should be two space indentation